lib/script_base.rb

```diff
@@ -54,6 +55,9 @@ def run
 
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
+    if result.messages && result.messages.length > 0
+      stderr.puts "*Messages*: #{result.messages.map { |message| "`#{message}`" }.join('\n')}"
```

Comment:
This will backticks the message *and* the UUID, should we start doing `[uuid, message]` tuples so we only backticks the UUID?

Reply:
also we had iterated with a bullet points per item, WDYT of that?

Reply:
Both comments addressed in latest commit.